org-code-javabuilder/lib/src/main/java/org/code/javabuilder/Util.java

```diff
@@ -2,18 +2,25 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.lang.reflect.Method;
 import java.net.URISyntaxException;
 import java.net.URL;
+import java.net.URLClassLoader;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Comparator;
+import java.util.List;
 import org.code.protocol.InternalErrorKey;
 import org.code.protocol.LoggerUtils;
 
 /** A set of static utility functions that are used in multiple locations */
 public class Util {
+  private Util() {
+    throw new UnsupportedOperationException("Instantiation of utility class is not allowed.");
+  }
+
```

Comment:
nice! can we make this a `final` class as well while we're here?

Reply:
done!